site/docs/platforms.md

```diff
@@ -156,6 +156,13 @@ $ bazel build --platforms=//:myplatform //...`
 $ bazel build --platforms=//:myplatform //:all`
 ```
 
+Note that incompatible tests in a [`test_suite`](be/general.html#test_suite)
+are similarly skipped if the `test_suite` is specified on the command line and
+[`--expand_test_suites`](command-line-reference.html#flag--expand_test_suites)
+is used. In other words, `test_suite` targets on the command line behave like
```

Comment:
Simplification suggestion: "on the command line with --expand_test_suites"?

Reply:
That's a good idea, thanks.
I applied your suggestion.